BoneCentral/Brain/Utilities/compassTools.js

```diff
@@ -0,0 +1,51 @@
+var vector = require('./vector.js');
+
+var getTrueCompass = function(compv, accelv) {
+    var av      = unitScale(accelv);
+    var right   = unitScale(vector.cross(av, compv));
+    var res     = unitScale(vector.cross(right, av));
+    return res;
+};
+
+var cdeg = function(compv, va) {
```

Comment:
Can we get these variables names updated to be clearer?  While I have an idea of what this function is doing, I still can't say that I know for sure.  I'm sure that updating the variable names and adding a commented description at the top of the function would really clear things up.